src/core/utils/types/multiselect-types.ts

```diff
@@ -0,0 +1,14 @@
+export type MultiselectOption = {
+  item: string;
+  value: string;
+};
+
+export type MultiselectExternalUpdateFunction = (filtersUpdate: {
+  key: string;
+  value: MultiselectOption[];
+}) => void;
+
+export type MultiselectExternalUpdate = {
+  key: string;
+  externalUpdateFunction: MultiselectExternalUpdateFunction;
+};
```

Comment:
Do we need these types outside of the multi select component? If not I think they should be moved to a file there.

Reply:
Noooo. 
1. I don't like type declarations in component files. 
2. The multiselect was made to be reused around other places too - so it makes sense to place the types in the general location too. That's how we agreed on it :))

Reply:
I did not mean to put the type in a component, but in a types.ts file in the multiselect folder alongside the component file. In that way it is self contained.

Reply:
Ohhh I see ❤️